makefile

```diff
@@ -41,8 +41,7 @@ endif
 ifeq ($(CYGWIN),1)
 	LIB_DIR:=$(shell cygpath -w $(LIB_DIR))
 endif
-LIB_DIR:=$(subst \,/,$(LIB_DIR))
-
+export LIB_DIR:=$(subst \,/,$(LIB_DIR))
```

Comment:
It is not recommended to change values and export in one cmd. Please separate the code into two lines.

Reply:
I'm just trying to keep the same style https://github.com/adoptium/TKG/blob/4c1094f7e8fdab2794a712c819beec812dc43757/makefile#L22